### PR TITLE
added option example

### PR DESCRIPTION
**Pull request by CCOSTAN**
it wasn't clear to me.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.cast.markdown [hunk 1]
```diff
@@ -23,6 +23,7 @@ Chromecast platform can also be forced to load by adding the following lines to
 # Example configuration.yaml entry
 media_player:
   - platform: cast
+    host: 192.168.1.10 #this is an optional variable
```
Comment (fabaff) [inline source/_components/media_player.cast.markdown]: We try to keep the configuration samples minimal. The `host` entry is mentioned a couple of line down the description of the variables.

Comment (CCOSTAN) [reply source/_components/media_player.cast.markdown]: agreed but as someone who just implemented it, it was not easily identifiable how to add the single variable.  at first, I tried indenting and then with a dash in a list..  There were no good examples.  

I understand the desire to keep the configuration minimal but for me, an example of one of the options was needed to understand the format.  This is also why I just added the single option to show the format.

Comment (CCOSTAN) [reply source/_components/media_player.cast.markdown]: @fabaff I can add a second 'advanced' example at the end of the page with both options if you think that would be cleaner for new users.

Comment (fabaff) [reply source/_components/media_player.cast.markdown]: Sure, we do that at a couple of places. If it helps then we should add it.

**Pull request merged**